src/components/Unauthenticated.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { FC, PropsWithChildren } from 'react';
+import { useFusionAuth } from '../providers/FusionAuthProvider';
+
+export const Unauthenticated: FC<PropsWithChildren> = props => {
```

Comment:
This component needs to be exported from `src/index.js` in order for it to be exported from the package (and available to consuming applications):

```ts
export * from './providers/FusionAuthProvider';
export * from './components/FusionAuthLoginButton';
export * from './components/FusionAuthLogoutButton';
export * from './components/FusionAuthRegisterButton';
export * from './components/RequireAuth';
export * from './components/Unauthenticated'; // add this line
export * from './components/withFusionAuth';
```